app/client/keybase/keyvault.go

```diff
@@ -0,0 +1,296 @@
+package keybase
```

Comment:
Can you create a package called `vault` at `app/client/keybase/vault`. If we ever need to add other backends in the future, it'll be more modular that way.